projects/project-metadata.json

```diff
@@ -6076,7 +6088,13 @@
           }
         ]
       },
-      "license": null,
+      "license": {
```

Comment:
@Aditi8007 As per the repository LICENSE folder, this project uses a BSD-3-Clause open source license. Please update that information in the license metadata section

Reply:
ok I will update